fe/fe-core/src/main/java/com/starrocks/alter/Alter.java

```diff
@@ -313,22 +313,22 @@ private void processModifyTableProperties(ModifyTablePropertiesClause modifyTabl
         int partitionTTL = INVALID;
         if (properties.containsKey(PropertyAnalyzer.PROPERTIES_PARTITION_TTL_NUMBER)) {
             partitionTTL = PropertyAnalyzer.analyzePartitionTimeToLive(properties);
-            properties.remove(PropertyAnalyzer.PROPERTIES_PARTITION_TTL_NUMBER);
```

Comment:
why

Reply:
I've checked that each `analyzePartitionTimeToLive ` method will remove the property. 

no need to remove it again.